assets/base.css

```diff
@@ -733,6 +733,25 @@ summary::-webkit-details-marker {
   outline: 0 !important;
 }
 
+.focus-offset:focus-visible {
+  outline: 0.2rem solid rgba(var(--color-foreground), 0.5);
+  outline-offset: 1rem;
+  box-shadow: 0 0 0 1rem rgb(var(--color-background)),
+    0 0 0.2rem 1.2rem rgba(var(--color-foreground), 0.3);
+}
+
+.focus-offset:focus {
+  outline: 0.2rem solid rgba(var(--color-foreground), 0.5);
+  outline-offset: 1rem;
+  box-shadow: 0 0 0 1rem rgb(var(--color-background)),
+    0 0 0.2rem 1.2rem rgba(var(--color-foreground), 0.3);
+}
```

Comment:
To ensure our `focus` CSS is valid for browsers that do not support `focus-visible`, we have to split them into separate selectors; otherwise the browser will ignore the entire comma-separated selector. 

[More details here, regarding a similar issue](https://css-tricks.com/dont-comma-separate-focus-within-if-you-need-deep-browser-support/)

Reply:
Oh interesting! Sounds good, thanks for the context.